tasks/install.redhat.yml

```diff
@@ -2,6 +2,6 @@
 
 - name: install Grafana package
   yum:
-    name: https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-4.4.2-1.x86_64.rpm
+    name: https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-5.0.4-1.x86_64.rpm
```

Comment:
Hey, can you make this use a variable, `grafana_rpm_url`, and default it in `defaults/main.yml`, thanks!